test/Dialect/FIRRTL/grand-central-taps.mlir

```diff
@@ -203,6 +222,7 @@ firrtl.circuit "TestHarness" attributes {
       { class = "firrtl.transforms.NoDedupAnnotation" }
     ],
     portAnnotations = [
+      [{class = "sifive.enterprise.grandcentral.ReferenceDataTapKey", id = 0 : i64, portID = 12 : i64, type = "portName"}],
```

Comment:
It seems like the fieldId should be here, and the annotation should refer to the wire symbol

Reply:
This would also be solved by lowering these scattered ID attributes to probe ops which would already be selecting the correct field and not need modification in the presence of type lowering or not.